components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/internal/IdpMgtListenerServiceComponent.java

```diff
@@ -33,6 +33,9 @@
  */
 public class IdpMgtListenerServiceComponent {
 
+    private IdpMgtListenerServiceComponent(){
```

Comment:
Don't use static methods and fields, use a dataholder, then this won't be suggested